README.md

```diff
@@ -6,13 +6,16 @@ API][arcgis-feature-service-rest-api].
 
 ## Table of Contents
 
-- [Features](#features)
-- [Install](#install)
-- [How to use](#how-to-use)
-- [How to query](#how-to-query)
-  - [Query Geometry](#query-geometry)
-  - [Query Properties](#query-properties)
-- [Site's `gatsby-node.js` example](#sites-gatsby-nodejs-example)
+- [gatsby-source-arcgis-feature-service](#gatsby-source-arcgis-feature-service)
+  - [Table of Contents](#table-of-contents)
```

Comment:
These headings shouldn't be part of the TOC.

Reply:
Yeah, probably autogenerated from my plugin. Removed.